config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -374,10 +419,11 @@ presubmits:
     - ^main$
     spec:
       containers:
-      - command:
-        - runner.sh
-        - ./scripts/ci-apidiff.sh
-        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210917-ee1e7c845b-1.21
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20210917-ee1e7c845b-1.21
+        command:
+          - runner.sh
+        args:
+          - ./scripts/ci-apidiff.sh
```

Comment:
we technically don't need the workaround for jobs that run locally like this one, but I don't think it hurts to make the change everywhere

Reply:
I robotically made the `iptables` change here, but then backed it out for runs of `ci-apidiff.sh` once I realized they aren't failing. So this is just a cosmetic change in the end.